build.sbt

```diff
@@ -576,13 +769,23 @@ lazy val compilerSettings = Seq(
           Seq(
             /*"-deprecation:false", */ "-Xlint:-unused,_"
           ) // unused-import breaks Circe Either shim
+
       case Some((2, m)) if m >= 12 =>
         base ++ Seq(
           "-deprecation:false",
           "-Xlint:-unused,_"
         ) // unused-import breaks Circe Either shim
+
       case Some((2, 11)) => base ++ Seq("-deprecation:false", "-Xlint", "-Ywarn-unused-import")
-      case _             => base ++ Seq("-Xlint")
+      case Some((2, _))  => base ++ Seq("-Xlint")
+      case _             => base
+    }
+  },
+  Test / scalacOptions ++= {
+    if (scalaBinaryVersion.value == "3") {
+      Seq("-Yretain-trees")
```

Comment:
Is this for debugging? Found this [page describing this option](https://docs.scala-lang.org/scala3/guides/migration/options-new.html) and was curious about its usage :)

Reply:
No, this strictly required to be able to resolve `Tree` for `Symbol` in Scala 3.
BTW the macro check whether it's present, otherwise it raises an error.